Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -70,6 +70,8 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
   // options
   groups.emplace_back(m_options = new ControlGroup(_trans("Options")));
   m_options->boolean_settings.emplace_back(
+      m_always_connected = new ControlGroup::BooleanSetting(_trans("Always Connected"), false));
```

Comment:
No. That's done separately (nowadays by me, usually weekly) and shouldn't be touched in PRs.